### PR TITLE
Fix directories being ignored.

### PR DESCRIPTION
**Pull request by LexLythius**
file.isNull holds true for directories, so they are ignored.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = function (user, group) {
 	var finalGid = typeof cachedGid === 'number' ? cachedGid : typeof group === 'number' ? group : null;
 
 	return through.obj(function (file, enc, cb) {
-		if (file.isNull()) {
+		if (file.isNull() && ! file.isDirectory()) {
```
Comment (kevva) [inline index.js]: `if (file.isNull() && !file.isDirectory()) {`

**Comment by sindresorhus** [comment]
@LexLythius Can you do the last commit https://github.com/LexLythius/gulp-chown/commit/4496e841567d726c1d28e7811725ad6bb61f7e30 as a separate pull request?

**Comment by LexLythius** [comment]
@sindresorhus I feel stupid, because that's what I tried to do. I committed f2326bf and 0a5944d, then placed the pull request, then committed 4496e84, not intended as part of the pull request. Github just tossed it into the same bag.

So how do I go about keeping them separate?

**Comment by sindresorhus** [comment]
This is why you should never do pull requests from master. Create new branches for each pull request.
https://help.github.com/articles/using-pull-requests/

Create a local branch called `tmp` (from master), go back to `master` branch and run `git reset --hard upstream/master` (<= must match the upstream you've set for this repo), now create a new branch (from master), cherry pick the first commit, then force push to the remote branch of this pull request. Then create another branch (from master), cherry pick the second commit, then publish the branch

**Comment by LexLythius** [comment]
Sent new, separate pull requests.

**Pull request closed without merging**